src/format.rs

```diff
@@ -70,7 +63,13 @@ impl fmt::Display for HumanDuration {
         try_unit!(60 * 60, "hour", "hours", "h");
         try_unit!(60, "minute", "minutes", "m");
         try_unit!(1, "second", "seconds", "s");
-        write!(f, "0{}", if alt { "s" } else { " seconds" })
+        if t >= 1500 {
```

Comment:
```suggestion
        if t >= 1500 { // 1500ms
```

Reply:
I found a less ambiguous way to make these comparisons, with more types :)